tests/php/ORM/HierarchyTest/TestObjectHideFromHierarchy.php.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace SilverStripe\ORM\Tests\HierarchyTest;
+
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\ORM\Hierarchy\Hierarchy;
+use SilverStripe\Versioned\Versioned;
+
+class TestObjectHideFromHierarchy extends TestObject implements TestOnly
```

Comment:
There is an existing test object HideTestObject used in different unit tests for testing $hide_from_hierarchy, however I couldn't use that class as it extends DataObject instead of TestObject